ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverPlugin.java

```diff
@@ -17,8 +17,6 @@
 package com.netflix.ndbench.plugin.cass;
 
 import com.datastax.driver.core.*;
```

Comment:
sent email to vinay w/ documentation updates (don't have perms to push to wiki myself). will cc u.